### ISSUE TITLE
react/jsx-no-target-blank false positive on multiple lines

### ISSUE THREAD

**Issue by valan** [yannickcr/eslint-plugin-react#1920]
`react/jsx-no-target-blank` triggers a warning even when the `ref` param is correct if the `ref` and the `target` are on separate lines

example:

```
<a
    target="_blank"
    ref="noopener noreferrer"
    href="https://github.com/FormidableLabs/electron-webpack-dashboard"
>
    webpack-dashboard
</a>
```

**Comment by ljharb** [comment]
That's `ref` - it should be `rel`.

**Comment by damianobarbati** [comment]
@valan did you solve with that? @ljharb probably I'm dumb but I really can't get why I still have the warning.

<img width="1021" alt="Screenshot 2020-01-18 at 10 51 07" src="https://user-images.githubusercontent.com/2225869/72661914-84b2f680-39e0-11ea-9362-db33b6b2276b.png">

**Comment by ljharb** [comment]
@damianobarbati if it works with a quote literal (instead of curly brackets), then I’d expect you need #2523 which is in v7.18.0. Please file a new issue if it’s still not working.

**Comment by damianobarbati** [comment]
Yes @ljharb it works with literal instead of brackets.
